packages/extension-chakra-storefront/src/pages/product-detail/index.jsx

```diff
@@ -62,6 +59,8 @@ const ProductDetail = () => {
     const toast = useToast()
     const navigate = useNavigation()
     const customerId = useCustomerId()
+    const {maxCacheAge: MAX_CACHE_AGE, staleWhileRevalidate: STALE_WHILE_REVALIDATE} =
```

Comment:
it appears `STALE_WHILE_REVALIDATE` is undefined?

Reply:
don't forget to update line 73

```diff
- `s-maxage=${MAX_CACHE_AGE}, stale-while-revalidate=${STALE_WHILE_REVALIDATE}`
+ `s-maxage=${MAX_CACHE_AGE}, stale-while-revalidate=${staleWhileRevalidate}`
```

Reply:
@kevinxh I renamed the value to have Uppercase. So we can keep them as MAX_CACHE_AGE and STALE_WHILE_REVALIDATE